server/src/schemas/typeDefs.ts

```diff
@@ -1,29 +1,34 @@
 const typeDefs = `
   type User {
-    _id: ID
+    _idUser: ID
     username: String
     email: String
     password: String
-    thoughts: [Thought]!
+    recipes: [Recipe!]!
   }
 
-  type Thought {
-    _id: ID
-    thoughtText: String
-    thoughtAuthor: String
-    createdAt: String
-    comments: [Comment]!
+  type Recipe {
+    _idRecipe: ID
+    recipeName: String
+    recipeAuthor: [User!]
```

Comment:
I assume we are referencing a single user, not an array of users? Also, see the conflict withe the recipe schema and way that this si being defined here (my note on line 30 of the src/models/Recipe.ts)